js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
 rolesPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 identityProvidersPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 groupsPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
+clientAuthorization=Autorisierung
+path=Pfad
+pairwiseSubAlgorithmSalt.label=Salz
```

IMHO "Salt" is a fixed term in hashing / encryption, and shouldn't be translated.

```suggestion
pairwiseSubAlgorithmSalt.label=Salt
```